### PR TITLE
Gcam usa doc

### PR DESCRIPTION
**Pull request by pralitp**
GCAM-USA documentation.  This is the version that is committed with GCAM 4.3.  Most of the GCAM USA documentation for papers have been for other versions (v3.2 or a research branch which has not yet been merged into the Core) so someone should take a look and ensure the text is accurate for this version.  I think perhaps @pkyle would have the best sense of this.

Still TODO is clean up the references.

### CODE REVIEW

**Comment by pkyle** [comment]
Hey @pralitp  I'm at meetings for the next couple of days so won't be able to look at this until Saturday. What I think I'll do for the review is to just make modifications directly to the branch, and push any changes directly up to the gcam-usa-doc branch--is that OK?

File: gcam-usa.md [hunk 1]
```diff
@@ -1,9 +1,201 @@
...
+
+<img src="gcam-figs/gcam-usa_CCS-curves.png" width="576" height="345" />
```
Comment (pkyle) [inline gcam-usa.md]: Note that the CCS curves figure has different grid region names than GCAM-USA. If possible we should switch them.

**Comment by pralitp** [comment]
@pkyle, that is fine.  Feel free to push commits directly.

File: gcam-usa.md (hunk 1 above)
Comment (pralitp) [reply gcam-usa.md]: Yea, those figures are old.  I'll try to dig up the source and recreate them.

**Pull request merged**